content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,133 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
+- [**Environments**](#environment): A project can have multiple environments. An environment typically maps to a _development stage_, like `dev`, `staging` or `prod`. **API keys are provisioned on the environment-level**.
+
+Here is a visual illustration of how these concepts relate to each other:
+
+![](/img/platform/pdp-concepts.png)
+
+### Account
+
+Your account is the basis for all interactions with PDP products. You can use it to manage your workspaces (and their projects).
+
+If you need to delete your account, go [here](/platform/support#deleting-your-pdp-account).
+
+### Workspace
+
+In your PDP account, you can create several workspaces. A workspace is an isolated space to host projects in which you use PDP products.
+
+In each workspace, you can:
+
+- view and manage all projects (and their environments) in that workspace.
+- manage billing, i.e. select a [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site), configure payment methods, or view the invoice history.
+- view the usage of your enabled PDP products across all projects in that workspace.
+- manage team members and change their roles / permissions in the workspace.
+
+### Project
+
+In each workspace, you can create several projects. A project typically represents an application (a product or service) your organization is offering. You typically have one [Prisma schema](/orm/prisma-schema/) per project.
+
+In each project, you can:
+
+- view and manage all environments in that project.
+
+The number of project you can create in a workspace depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in that workspace:
+
+Here's an overview of the number of projects per workspace on each plan:
+
+|              | **Starter** | **Pro** | **Business** | **Enterprise** |
+| ------------ | ----------- | ------- | ------------ | -------------- |
+| **Projects** | 5           | 10      | 15           | Custom         |
+
+### Environment
+
+An environment is an isolated space to provision PDP products for a given project. Environments typically map to development stages, like `dev`, `staging` or `prod`.
+
+In each environment, you can:
+
+- enable, disable and manage PDP products (Pulse, Accelerate, ...).
+- for **Accelerate**:
+  - set your database connection string.
+  - generate API keys.
+  - change the connection pool size.
+  - enable static IPs.
+- for **Pulse**:
+  - set your database connection string (of a [Pulse-compatible database](/pulse/database-setup)).
+  - configure the _region_ where Pulse is running.
+  - configure database replication.
+  - generate API keys.
+
+The number of environments you can create in a project depends on the [pricing plan](https://pris.ly/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in your workspace:
```

Comment:
Same comment here about the prisma.io vs. pris.ly link & mentioning the specific numbers of envs allowed per plan